packages/itwinui-react/src/core/Tree/Tree.tsx

```diff
@@ -334,6 +340,7 @@ export const Tree = <T,>(props: TreeProps<T>) => {
           onKeyDown={handleKeyDown}
           onFocus={handleFocus}
           className={className}
+          data-iui-size={size}
```

Comment:
When `size` of `"default"` is passed, `data-iui-size` is set to `default`. But in the HTML demo, `data-iui-size` is not set for the default size.

This doesn't affect any styles. But to confirm, is setting this attribute when `size="default"` alright?

Reply:
doesn't matter too much, but we can do this:
```suggestion
          data-iui-size={size === "small" ? "small" : undefined}
```